.github/workflows/node.js.yml

```diff
@@ -10,12 +10,41 @@ on:
     branches: [ "main" ]
 
 env:
+  # Required for front-end operation.
```

Comment:
You have overcomplicated it a bit here. All we need to be able to do in the pipeline is manually trigger it to run the tests against the hosted version in AWS. So you could simplify this right down.

Reply:
Made changes so that the AWS site is used, and that the UI and accessibility tests run automatically after merging to main.